lib/dry/core/class_attributes.rb

```diff
@@ -80,10 +80,8 @@ def defines(*args, type: ::Object, coerce: IDENTITY)
               if Undefined.equal?(value)
                 if instance_variable_defined?(ivar)
                   instance_variable_get(ivar)
-                else
-                  nil
```

Comment:
I disagree with rubocop on this. The line explicitly returns a concrete value rather than relying on implicit ruby behavior. In my view, rubocop's rule must be the opposite of this one. It's better for coverage, it's better for reading the intention. I triggered because already saw it somewhere else and I still disagree :) I don't mind committing it, it's just a rant ;)

Reply:
@flash-gordon restored with 4789de4